src/components/Theme/src/default-theme.js

```diff
@@ -20,6 +20,24 @@ export default function defaultTheme() {
 			baseSize: 16,
 			sizeScale: 1.17,
 		},
+		shapes: {
+			squared: {
+				defaultBorderRadius: '0px',
+				buttonBorderRadius: '0px',
+				imageBorderRadius: '0px',
+			},
+			rounded: {
+				defaultBorderRadius: '4px',
+				buttonBorderRadius: '8px',
+				imageBorderRadius: '16px',
+			},
+			pill: {
+				defaultBorderRadius: '4px',
+				buttonBorderRadius: '100px',
+				imageBorderRadius: '16px',
+			},
```

Comment:
Could all the local shape options support a string border radius and an alias like `pill`?

Reply:
All of the individual component shapes still work. So you can leave the main shapes prop alone and just reference a different shape  for a given component. It's just that by default it passes the global shape.
```
		button: {
			shape: 'pill',
		},
		card: {
			shape: 'squared',
		},
```
would both still work.

Reply:
Re: redefining our aliases. I don't really see an issue with it -  no developer ever needs to change them and we can leave it out of the theme docs. Out of the box nobody needs to pass anything to MTheme besides:

```
shapes: {
    global: 'pill',
}
```

for Website, where we have a separate control for images, it would be:
```
shapes: {
    global: 'pill',
},
image: {
    shape: 'squared',
}
```
The only reason we'd ever expose the fact that the shapes are aliased and could be overridden is if there was a use case that warranted it.